### ISSUE TITLE
After swipe animation blocks second swipe

### ISSUE THREAD

**Issue by jand000** [kenwheeler/slick#1060]
When swiping on Smartphone and releasing the finger, the slide completes the animation. During this time no swiping is possible. This prevents to swipe fast through a slideshow.

**Comment by fredericpfisterer** [comment]
I'm having the same problem. Any ideas how to change this behaviour ? Could we have a free scroll option with inertia on touch screen ?

**Comment by simeydotme** [comment]
@fredericpfisterer you mean a div with `overflow: scroll`?

**Comment by fredericpfisterer** [comment]
@simeydotme is right. But I would love to be able to use slick's centerMode and Infinite looping as well. I added swipeToSlide and the feeling on tablet is already better. It's still missing something to feel natural though.

**Comment by simeydotme** [comment]
i believe it's being looked at for 2.0?

**Comment by fredericpfisterer** [comment]
Good to hear ! Where can I get more info about 2.0 ?

**Comment by simeydotme** [comment]
Actually it is just hearsay from a conversation I had, actually I raised the idea of inertia scrolling with a "lazy-stick" .. IE: after inertia diminishes it would stick to the closest slide :smile:

So my previous message is actually a question for @kenwheeler :P

**Comment by fredericpfisterer** [comment]
It's exactly what I as looking for. I know swiperJs has it but we had problems with it and angularJs. I guess I'll wait then. Thanks for your time. Cheers.

**Comment by kenwheeler** [comment]
The inertial scrolling doesn't really work with the way slick is architected. 2.0 is going to do things a bit differently to allow for this and other flexibilities.

**Comment by fredericpfisterer** [comment]
Any idea when we could get our hands on it ? anyway keep up the good work.

**Comment by kenwheeler** [comment]
Targeting springish. Got a lot going on at the FT in the near future.

**Comment by desandro** [comment]
You might want to try [Flickity](http://flickity.metafizzy.co). Flickity uses physics-based animation, so inertial scrolling is already built in. You can enable [`freeScroll`](http://flickity.metafizzy.co/options.html#freescroll) which allows the slider to be positioned without aligning to a cell.

**Comment by dallanlee** [comment]
Not allowing the user to swipe before the animation completes, therefore limiting the navigation to only one slide at a time, is the ONLY reason I am not using this carousel in all my projects. So unfortunate! Could we get this changed???

**Comment by dallanlee** [comment]
@akashnigam010 I agree. I switched to Flickity a while ago and use it on all my projects.
